tests/profiles/googlefonts_test.py

```diff
@@ -3150,50 +3151,95 @@ def test_check_vertical_metrics_regressions(cabin_ttFonts):
                 api_gfonts_ttFont,
                 style,
                 remote_styles,
+                regular_ttFont,
+                regular_remote_style,
                 metadata_file,
                 family_metadata)
-    from copy import copy
+    from copy import deepcopy
+
+    remote = regular_ttFont([TTFont(f) for f in cabin_fonts])
+    ttFont = regular_ttFont([TTFont(f) for f in cabin_fonts])
+
+    # Cabin test family should match by default
+    assert_PASS(check(ttFont, remote),
+                'with a good family...')
+
+    # FAIL with a changed vertical metric value
+    remote2 = deepcopy(remote)
+    ttFont2 = deepcopy(ttFont)
```

Comment:
@m4rc1e, I've seen you using `deepcopy` in the past just like you do here. I am not sure this is actually needed, though. My guess is that by reusing a `ttFont` variable and assigning to it a new instance of a TTFont object, the previous instance would be destroyed as python's garbage collection would delete it. Are you sure we need `deepcopy`? Or maybe there's more in the way it behaves that I am not fully aware of. If so please let me know.

Reply:
Plain ol `copy` doesn't work on TTFont objects. I guess could just reinstantiate the TTFonts but I think this is more cumbersome than just calling `deepcopy`

https://docs.python.org/3/library/copy.html#:~:text=A%20shallow%20copy%20constructs%20a,objects%20found%20in%20the%20original.

Reply:
Just to double check, I replaced `deepcopy` with `copy` and it didn't work.